FRAuth/FRAuth/Callbacks/DeviceBindingCallback.swift

```diff
@@ -0,0 +1,254 @@
+// 
+//  DeviceBindingCallback.swift
+//  FRAuth
+//
+//  Copyright (c) 2022 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+import JOSESwift
+
+/**
+ * Callback to collect the device binding information
+ */
+open class DeviceBindingCallback: MultipleValuesCallback {
+    
+    //  MARK: - Properties
+    
+    /// The userId received from server
```

Comment:
can you check the Android changes for this , all this needs to be private set and public get i belive ?